en/flight_controller/cuav_v5_plus.md

```diff
@@ -71,16 +71,16 @@ Download **V5+** pinouts from [here](http://manual.cuav.net/V5-Plus.pdf).
 ## Voltage Ratings
 
 *V5+ AutoPilot* can be triple-redundant on the power supply if three power sources are supplied.
```

Comment:
@davids5 In the nano I changed this to remove the double redundant power info story - since you can only use `Power` during flight. My take is that for v5+ this is also true - you would only use `power1` or `power2` (even though in theory you could also use USB, in reality you would not).

Upshot, this should change to "double redundant" right?

Reply:
redundant is all it should say.

Reply:
@hamishwillee  rails is the wrong word. Use Power sources. The X can be powered from any or all of the 3 Power sources.

Reply:
Thanks. The whole section is updated, and now renders like this:

![image](https://user-images.githubusercontent.com/5368500/61921043-19dbb200-af9f-11e9-89ee-8bb7c2a4b590.png)

Reply:
The text in blue - make no sense. "On FMUv5 based FMUs with PX4IO module, as is the case for the V5+ FMU. The Servo Power Rail is only monitored by the FMU. It is neither powered by, nor provides power to the FMU. However, the pins marked + are all common and a BEC may be connected to any of the servo pins sets to power the servo power rail."

Reply:
Thanks. Fixed as per https://github.com/PX4/px4_user_guide/pull/538/commits/7166af51c69af140a46ee4c27cec35fd5781472e